src/store/check-font-loaded.js

```diff
@@ -1,4 +1,4 @@
-import { UPDATE_FONT_LOADED } from '../../actions';
+import { UPDATE_FONT_LOADED } from '../actions';
 
 /**
  * Prevent chart from displaying until the webfont has loaded,
```

Comment:
I feel like this file shouldn't live in the store as it's not related to the store, maybe utils would be a better place?

Reply:
Sure 🤷‍♂